### PR TITLE
docs: polish  readme file

### PR DESCRIPTION
**Pull request by zmsn-2077**
# Description

Describe your changes in detail.

## Motivation and Context

Why is this change required? What problem does it solve?
If it fixes an open issue, please link to the issue here.
You can use the syntax `close #1314520` if this solves the issue #15213

- [ ] I have raised an issue to propose this change ([required](https://github.com/PKU-Alignment/align-anything/issues) for new features and bug fixes)

## Types of changes

What types of changes does your code introduce? Put an `x` in all the boxes that apply:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds core functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentation (update in the documentation)

## Checklist

Go over all the following points, and put an `x` in all the boxes that apply.
If you are unsure about any of these, don't hesitate to ask. We are here to help!

- [X] I have read the [CONTRIBUTION](https://github.com/PKU-Alignment/align-anything/blob/HEAD/.github/CONTRIBUTING.md) guide. (**required**)
- [ ] My change requires a change to the documentation.
- [ ] I have updated the tests accordingly. (*required for a bug fix or a new feature*)
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -124,7 +124,7 @@ export CUDA_HOME=$CONDA_PREFIX
 export CUDA_HOME="/usr/local/cuda"
 ```
 
-Fianlly, install this project by:
+Fianlly, install `align-anything` by:
```
Comment (Gaiejj) [inline README.md:127]: Maybe we can unify it to Align-Anything

Comment (zmsn-2077) [reply README.md:127]: from xuehai: the name of the module is best in lowercase.

Comment (zmsn-2077) [reply README.md:127]: unify to align-anything

File: README.md [hunk 2]
```diff
@@ -82,10 +82,10 @@ We support evaluation datasets for `Text -> Text`, `Text + Image -> Text` and `T
```
Comment (cby-pku) [inline README.md:82]: What about using `under construction` or `coming soon` other than `in the planning`

File: README.md (hunk 1 above)
Comment (cby-pku) [reply README.md:127]: Nice advice ! But conda env name is also `align-anything`, so maybe `align-anything` is better.

File: README.md [hunk 3]
```diff
@@ -301,10 +301,6 @@ python3 -m align_anything.serve.arena \
 
 <img src="assets/arena_demo.gif" alt="arena_demo" style="width:600px;">
 
-## Why do we open source align-anything?
-
-Ensuring that the behavior of AI system aligns with human intentions and values is crucial, and alignment techniques provide an effective solution. For large language models (LLMs), methods such as reinforcement learning with human feedback (RLHF) and direct preference optimization (DPO) have significantly improved performance and safety. As models evolve to handle any-modality inputs and outputs, effectively aligning them remains a current research challenge. Align-Anything framework integrates alignment tuning across modalities using well-designed interfaces and advanced abstractions, offering a comprehensive testbed for research.
-
 ## Report Issues
 
 If you have any questions in the process of using Align-Anything, don't hesitate to ask your questions on [the GitHub issue page](https://github.com/PKU-Alignment/align-anything/issues/new/choose), we will reply to you in 2-3 working days.
```
Comment (cby-pku) [inline README.md:306]: Well, suggest unifiy this `Align-Anything`, either `Align-Anything` or `align-anything`. Please check this term through the document and README.

**Review by cby-pku — COMMENTED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (Gaiejj) [reply README.md:127]: Thank you for the quick response! I think this has addressed most of my concerns.

**Review by Gaiejj — APPROVED** [review]
LGTM.

File: README.md [hunk 4]
```diff
@@ -301,13 +301,9 @@ python3 -m align_anything.serve.arena \
...
 
-## Why do we open source align-anything?
-
-Ensuring that the behavior of AI system aligns with human intentions and values is crucial, and alignment techniques provide an effective solution. For large language models (LLMs), methods such as reinforcement learning with human feedback (RLHF) and direct preference optimization (DPO) have significantly improved performance and safety. As models evolve to handle any-modality inputs and outputs, effectively aligning them remains a current research challenge. Align-Anything framework integrates alignment tuning across modalities using well-designed interfaces and advanced abstractions, offering a comprehensive testbed for research.
-
 ## Report Issues
 
-If you have any questions in the process of using Align-Anything, don't hesitate to ask your questions on [the GitHub issue page](https://github.com/PKU-Alignment/align-anything/issues/new/choose), we will reply to you in 2-3 working days.
+If you have any questions in the process of using align-Anything, don't hesitate to ask your questions on [the GitHub issue page](https://github.com/PKU-Alignment/align-anything/issues/new/choose), we will reply to you in 2-3 working days.
```
Comment (Gaiejj) [inline README.md:306]: align-Anything

File: README.md [hunk 5]
```diff
@@ -303,7 +303,7 @@ python3 -m align_anything.serve.arena \
 
 ## Report Issues
 
-If you have any questions in the process of using Align-Anything, don't hesitate to ask your questions on [the GitHub issue page](https://github.com/PKU-Alignment/align-anything/issues/new/choose), we will reply to you in 2-3 working days.
+If you have any questions in the process of using align-Anything, don't hesitate to ask your questions on [the GitHub issue page](https://github.com/PKU-Alignment/align-anything/issues/new/choose), we will reply to you in 2-3 working days.
```
Comment (cby-pku) [inline README.md:306]: Oh, minor bugs

Comment (Gaiejj) [reply README.md:306]: additionally, may be `align-anything` is better since we have used it before

**Review by cby-pku — APPROVED** [review]

**Pull request merged**